src/encoded/commands/deploy copy.py

```diff
@@ -0,0 +1,183 @@
+import boto3
```

Comment:
Delete this file, don't check in copies.  Git has all history of all changes, trust it!!!